[EN/ZH] Add `Chinese Charting Contest 2024` (#12116)
ppy/osu-wiki: wiki/Contests/CCC/2024/en.md

## Song List

| Song | Length | BPM |
| :-- | :-- | :-- |
| you - Texture Garden | 02:28 | 128 |
| Kirara Magic & Ulchero - Online Fantasy (feat. Moon Jelly) | 03:45 | 126 |
| Essbee - Rebel Of Wildernes | 02:28 | 166 |
| mybear - Last Slurp | 03:10 | 126 |
| bobawhale & glitchless - starsignal! | 02:52 | 170 |
| awfuless - Temptation | 03:04 | 173 |

## Results

| Place | Mapper(s) |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") Winner | ::{ flag=CN }:: [BKwind](https://osu.ppy.sh/users/8900975) |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") 2nd place | ::{ flag=CN }:: [\_Yiiiii](https://osu.ppy.sh/users/6066359) |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") 3rd place | ::{ flag=CN }:: [tyrcs](https://osu.ppy.sh/users/13026904) |
| Participation Award (4th-7th) | ::{ flag=CN }:: [Boomushroom](https://osu.ppy.sh/users/10588550), ::{ flag=CN }:: [Seiran-](https://osu.ppy.sh/users/14351534), ::{ flag=RU }:: [FlappiBakuse](https://osu.ppy.sh/users/11724295), ::{ flag=CN }:: [wIntEr7z](https://osu.ppy.sh/users/16509588) |

## Ruleset

### Contest rules

> - **This contest is for the `osu!mania` mode only.**

Reviewer: ```suggestion
- **This contest is limited to the `osu!mania` mode only.**
```

> - **This contest is for Chinese players only, including players with the flags of ::{ flag=CN }:: Chinese mainland, ::{ flag=HK }:: Hong Kong, ::{ flag=MO }:: Macao, and ::{ flag=TW }:: Taiwan eligible for participation.** If your flags aren't those four but actually it, you can participate this contest if player can prove that.

Reviewer: ```suggestion
- **This contest is exclusively for Chinese players, including those with the flags of ::{ flag=CN }:: Chinese mainland, ::{ flag=HK }:: Hong Kong, ::{ flag=MO }:: Macao, and ::{ flag=TW }:: Taiwan.** If a player's flag is not one of the four listed but they are still eligible, participation is allowed with proof of eligibility.

```

> - **Only account valid for logging into the [Mappers' Guild](https://mappersguild.com) are allowed.**

Reviewer 2: since the submission need to be submitted by mg website, so if the account can't log in mg then the player can't participate the contest, that's what I mean and make a information like that.

- **Submissions must be in the `.osz` format.**
- **The osu! client must be able to load and read your submission.**
- **Submission must be `4K`, `5K`, `6K` or `7K`.** Other key modes are not considered in this contest (including n+1 playstyle).

> - **Beatmaps' mapping procress should be based on [Ranking Criteria.](/wiki/Ranking_criteria/osu!mania)** There are no requirements to follow RC compulsory, but the beatmap needs to be used Ranking Criteria as a fundamental reference, or that will effect your final score.

Reviewer: ```suggestion
- **Beatmap mapping process should be based on [Ranking Criteria](/wiki/Ranking_criteria/osu!mania).** While it is not compulsory to follow the Ranking Criteria strictly, the beatmap should use the Ranking Criteria as a fundamental reference, or this will affect the final score.

```

> - **Videos, Skins, Storyboards and Hitsounds are NOT allowed.**

Reviewer 2: all, since any tiny changes in file will cause the osz. different with original setting, that's probably can be a clues of cheating

- **Title, Artist, MP3, Source, Tags, Background and Preview Point must NOT be modified.**
- **Submissions must be created solely by the participant.** Collaborations are not allowed for this contest.

> - **Submissions must NOT be uploaded to the osu! website until after the contest's results are finalised.**

Reviewer 2: anything need to be changed?

> - **Submissions must NOT be uploaded to the osu! website until after the contest's results are finalised.**

Reviewer 3: probably not since he didn't respond
